libkbfs/key_manager_test.go

```diff
@@ -1638,6 +1670,14 @@ func testKeyManagerRekeyBit(t *testing.T, ver MetadataVer) {
 		t.Fatalf("Device 2 couldn't read a: %+v", err)
 	}
 
+	// Cancel the original promptPaper timer for user2 now that the
+	// TLF is readable; otherwise the rekey later on will hang because
+	// it looks like prompt for paper was already set.  TODO: should
```

Comment:
I guess stopping when you see a rekey from another device is fine, because if another one is needed, presumably that device will check for it again in 10s.  I'll file a nonurgent ticket for this.